doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -432,7 +443,45 @@ and ``coqtop``, unless stated otherwise:
 :-list-tags: Print the highlight tags known by Coq as well as their
   currently associated color and exit.
 :-h, --help: Print a short usage and exit.
+:-time: Output timing information for each command to standard output.
+:-time-file *file*: Output timing information for each command to the given file.
+:-profile *file*: Output :ref:`profiling` information to the given file.
+
+.. _profiling:
+
+Profiling
+---------
+
+Using the `coqc` command line argument `-profile` or the environment
+variable `PROFILE` with `coq_makefile`, profiling information is produced in
+`Google trace format <https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/edit>`.
+
+The output consists of duration events for the execution of various
+components of Coq (for instance `process` for the whole file,
+`command` for each command, `pretyping` for elaboration).
```

Comment:
```suggestion
The output gives the duration and event counts for the execution of
components of Coq (for instance `process` for the whole file,
`command` for each command, `pretyping` for elaboration).
```

Maybe mention that some categories are included in others, eg command always includes parse_command time.

Reply:
>Maybe mention that some categories are included in others, eg command always includes parse_command time.

This could change in the future.